packages/jest-haste-map/src/crawlers/watchman.js

```diff
@@ -116,7 +134,7 @@ module.exports = async function watchmanCrawl(
             ? // Use the `since` generator if we have a clock available
               {expression, fields, since: clocks.get(root)}
             : // Otherwise use the `glob` filter
-              {expression, fields, glob};
+              {expression, fields, glob, glob_includedotfiles: true};
```

Comment:
I've seen `dotfiles` are tracked now; what's the reason for this?

Reply:
To support [PNPM](https://github.com/pnpm/pnpm), which creates hidden directories (eg: `.registry.npmjs.org`, `.github.com`, etc).